etc/frontend/di.xml

```diff
@@ -24,13 +24,23 @@
     <type name="Adyen\ExpressCheckout\Block\GooglePay\Shortcut\Button">
         <arguments>
             <argument name="data" xsi:type="array">
-                <item name="template" xsi:type="string">Adyen_ExpressCheckout::googlepay/shortcut.phtml</item>
+                <item name="template" xsi:type="string">Adyen_ExpressCheckout::abstract/shortcut.phtml</item>
                 <item name="alias" xsi:type="string">adyen.googlepay.mini-cart</item>
                 <item name="button_id" xsi:type="string">adyen-googlepay-mini-cart</item>
             </argument>
             <argument name="payment" xsi:type="object">AdyenPaymentHppFacade</argument>
         </arguments>
     </type>
+    <type name="Adyen\ExpressCheckout\Block\Paypal\Shortcut\Button">
+        <arguments>
+            <argument name="data" xsi:type="array">
+                <item name="template" xsi:type="string">Adyen_ExpressCheckout::abstract/shortcut.phtml</item>
+                <item name="alias" xsi:type="string">adyen.paypal.mini-cart</item>
+                <item name="button_id" xsi:type="string">adyen-paypal-mini-cart</item>
+            </argument>
+            <argument name="payment" xsi:type="object">AdyenPaymentHppFacade</argument>
```

Comment:
`AdyenPaymentHppFacade` facade is deprecated. What do you think about migrating this to `AdyenPaymentPaypalExpressFacade`?

Similarly, other payment method methods can use their own facades like `AdyenPaymentGooglepayFacade` or `AdyenPaymentApplePayFacade`.